suite-native/atoms/src/Input/Input.tsx

```diff
@@ -33,67 +42,96 @@ type InputWrapperStyleProps = {
     hasWarning: boolean;
     hasError: boolean;
     isLabelMinimized: boolean;
+    colorScheme: InputColorScheme;
+};
+
+const inputTextColorSchemeStyles: Record<InputColorScheme, NativeStyleObject> = {
```

Comment:
I don't understand what are you actually doing with colors here, this should not be necessary color name should be always same in all color schemes, doing this specific rewrites is way to hell with themes. If not it's not looks good it's not ours fault but because Ondra defined colors bad.

He is actually working on better color pallete right now, with new names where this definitely shoudn't be necessary, so please remove this.

Reply:
The thing is there are screens prepared for onboarding and scanning with inputs and colors we don't have defined currently. So I have added it to fulfill Figma designs.

<img width="246" alt="Snímek obrazovky 2022-08-04 v 16 36 51" src="https://user-images.githubusercontent.com/2227592/182874541-adbaef75-6612-45c3-8612-02e06450ff7f.png">